client/lib/signup/cart.js

```diff
@@ -13,12 +13,17 @@ import wpcom from 'lib/wp';
 import productsListFactory from 'lib/products-list';
 const productsList = productsListFactory();
 import { cartItems, fillInAllCartItemAttributes } from 'lib/cart-values';
+import getInitialQueryArguments from 'state/selectors/get-initial-query-arguments';
 
-function addProductsToCart( cart, newCartItems ) {
+function addProductsToCart( cart, newCartItems, reduxStore ) {
+	const initialQueryArguments = getInitialQueryArguments( reduxStore.getState() );
```

Comment:
We should avoid this and extract whatever is needed in `step-actions`. In theory we should never have to pass `reduxStore` explicitly.